gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -230,6 +232,58 @@ get_mode (void)
     return GIS_DRIVER_MODE_EXISTING_USER;
 }
 
+static gboolean
+launch_tutorial_cb (GMainLoop *main_loop)
+{
+  FBERemote *fbe_remote;
+  char *language;
+  GError *error = NULL;
+
+  fbe_remote = fberemote_proxy_new_for_bus_sync (G_BUS_TYPE_SESSION,
+                                                 G_DBUS_PROXY_FLAGS_NONE,
+                                                 "com.endlessm.Tutorial",
+                                                 "/com/endlessm/Tutorial/FBERemote",
+                                                 NULL, &error);
+
+  if (error != NULL)
+    {
```

Comment:
Ah, this snuck in because I originally copied it from the summary page file, which uses inconsistent indentation.